### PR TITLE
Systematize additive scenes

### LINKED ISSUE #55: Rearrange LevelManager objects for fool proof no-code level building
This branch will copy new_game_plus into a new scene and rearrange the gameobjects' hierarchies, probably some minor changes will be made in the level objects' scripts too to accustom to the changes.

### PR DESCRIPTION
**Pull request by TizioMaurizio**
Fixed all known bugs, all features should be there, starting from 0_Main you can cycle through Area_00, Area_01 and Area_00 again by completing all the puzzles in each area.

### CODE REVIEW

File: Assets/Code/Scripts/NoclipRealityManagement/RealityMovement.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ private enum MovementState       // define player states
     [SerializeField] private float _gravityMultiplier = 1f;
     [SerializeField] private Transform _orientation;
 
-    private float _gravity = 9.81f;  // This is used for the movement force 
+    [SerializeField] private float _gravity = 9.81f;  // This is used for the movement force //Questo peccato al cospetto di Dio va fixato
```
Comment (stefanofossati) [inline Assets/Code/Scripts/NoclipRealityManagement/RealityMovement.cs:56]: ?

**Review by stefanofossati — COMMENTED** [review]
All works from my side. Maybe it's better to write the needed configuration to run all correctly

File: Assets/Code/Scripts/NoclipRealityManagement/RealityMovement.cs (hunk 1 above)
Comment (TizioMaurizio) [reply Assets/Code/Scripts/NoclipRealityManagement/RealityMovement.cs:56]: ahhhhhhh gravity, such a controversial topic
The problem here is that we should either use Physics.gravity or manage this variable properly, right now it is used too ambiguously. Anyway here it was changed just to test the level, the matter has to be managed on the other branch

Comment (spig95) [reply Assets/Code/Scripts/NoclipRealityManagement/RealityMovement.cs:56]: As agreed, we removed this part. So we do not mix changes to the controller in different PRs

**Comment by TizioMaurizio** [comment]
IMPORTANT the gravity fix was because as of now at each scene change gravity is remultiplied by the player

**Comment by TizioMaurizio** [comment]
I fixed gravity again because even if we happen to have a conflict it is the only healthy solution to this problem.
Anyway now the sequence Area_00, Area_01, Area_00 is properly executed.
In order to properly execute from the menu Menu_0 should avoid killing 0_Main @bottolo

**Review by spig95 — APPROVED** [review]

**Pull request merged**